metro.config.js

```
@@ -1,13 +1,13 @@
 const { getDefaultConfig, mergeConfig } = require("@react-native/metro-config");
+const { getDefaultConfig, mergeConfig } = require("@react-native/metro-config");
```

Biome (2.1.2)

[error] Shouldn't redeclare 'getDefaultConfig'. Consider to delete it or rename it.

'getDefaultConfig' is defined here:


(lint/suspicious/noRedeclare)